src/directional_position.rs

```diff
@@ -79,7 +75,7 @@ impl<'a> Stream<'a> {
 
         if self.starts_with(b"left") {
             self.advance(4);
-            return Ok(DirectionalPosition::Left);
+            Ok(DirectionalPosition::Left)
         } else if self.starts_with(b"right") {
             self.advance(5);
             return Ok(DirectionalPosition::Right);
```

Comment:
:thinking:

Reply:
Yeah, seen this before. Have not reported upstream.